roombasim/pittras/ai/hold_position_demo_controller.py

```diff
@@ -0,0 +1,23 @@
+'''
+hold_position_demo_controller.py
+'''
+
+from roombasim.ai import Controller
+
+def hold_position_task_completion_callback(status):
```

Comment:
the task completion signature is (status, message(optional)) but callbacks need to at least accept the message, so should be:

`def hold_position_task_completion_callback(status, message='')`

Reply:
actually, the callback is called internally and will always have the message parameter, so the signature can just be `(status, message)`